Sources/protoc-gen-swift/MessageFieldGenerator.swift

```diff
@@ -52,6 +52,26 @@ extension Google_Protobuf_FieldDescriptorProto {
         let m = context.getMessageForPath(path: typeName)!
         return m.options.mapEntry
     }
+    // Returns true/false if the field will be hold a Message. map<> is the
+    func getFieldHoldsMessage(context: Context) -> Bool {
```

Comment:
Is there a reason this is a separate function, instead of putting the body in the property below that calls it? That would get rid of the awkward name.